codespell_lib/_codespell.py

```diff
@@ -280,7 +285,7 @@ def parse_options(args):
                              'patterns to ignore by treating as whitespace. '
                              'When writing regexes, consider ensuring there '
```

Comment:
Can `regexes` be updated to `regular expressions`?

Reply:
Done